deploy/clowdapp.yaml

```diff
@@ -186,6 +188,7 @@ objects:
           name: ccx_messaging.downloaders.http_downloader.HTTPDownloader
           kwargs:
             max_archive_size: 100MiB
+            allow_unsafe_links: $ALLOW_UNSAFE_LINKS
```

Comment:
This may not work. You need to use `${}` notation. Have you checked in ephemeral?

Reply:
Fixed just to be safe. It should work anyway, because this way it is used in ccx-data-pipeline.